tests/unit/modules/win_firewall_test.py

```diff
@@ -28,7 +28,7 @@
 IS_WIN = win_firewall.__virtual__()
 
 
-@skipIf(not IS_WIN, "This test case runs only on Windows system")
+@skipIf(not IS_WIN[0], "This test case runs only on Windows system")
```

Comment:
This will just break in the other direction. The virtual function will return a single boolean if it is Windows.